app/models/challenge_signup_summary.rb

```diff
@@ -38,6 +51,85 @@ def tag_summary(tag)
     end
   end
 
+  public
+
+  ######################################################################
+  # GENERATING THE SUMMARY IN RESQUE
+  ######################################################################
+
+  @queue = :collection
+
+  # The action to be performed when this class is enqueued using Resque.
+  def self.perform(collection_id)
+    collection = Collection.find(collection_id)
+    summary = ChallengeSignupSummary.new(collection)
+    summary.generate_in_background
```

Comment:
Is it worth scheduling another run in say 50 minutes if the challenge is still open ?

Reply:
I'm not sure I understand what you're suggesting. Do you want me to make sure that it's consistently regenerated every 50 minutes until sign-ups close, even if no one is looking at it?